src/components/KMultiselect/KMultiselect.vue

```diff
@@ -165,9 +156,18 @@
                 </template>
               </KMultiselectItem>
               <KMultiselectItem
-                v-if="!sortedItems.length && !$slots.empty"
+                v-if="enableItemCreation && uniqueFilterStr"
+                key="k-multiselect-new-item"
+                class="k-multiselect-empty-item"
+                data-testid="k-multiselect-add-item"
+                :item="{ label: `${filterStr} (New value)`, value: 'add_item' }"
```

Comment:
This is the item you click to trigger the creation of a new item. It can have whatever value, it doesn't get used anywhere (it isn't the value used if they do create the item).